crates/next-core/src/next_font/google/util.rs

```diff
@@ -114,48 +152,66 @@ pub(super) fn get_stylesheet_url(
 ) -> Result<String> {
     // Variants are all combinations of weight and style, each variant will result
     // in a separate font file
-    let mut variants: Vec<Vec<(&str, &str)>> = vec![];
-    if axes.wght.is_empty() {
+    let mut variants: Vec<Vec<(&str, VariantValue)>> = vec![];
+
+    let weights = match &axes.wght {
+        FontAxesWeights::Variable(Some(wght)) => {
+            vec![VariantValue::String(wght.to_owned())]
```

Comment:
Nit: This is an `&RcStr`. `to_owned()` implies some sort of conversion (e.g. `&str` to `String`), which this is not.

```suggestion
            vec![VariantValue::String(wght.clone())]
```